Bottom: 0f98c38cace5fce9c248cccc2bc010c87014538f
Top:    9f140e920904af6b8c3b761518e0959b4ee14683
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-23 22:13:44 +0000

Refresh of off-to-find-burried-treasure-o

---

diff --git a/src/find.c b/src/find.c
index 56186b7..6a00e8b 100644
--- a/src/find.c
+++ b/src/find.c
@@ -4008,7 +4008,7 @@ IsPolygonAnIsland (LayerType *layer, PolygonType *polygon)
    */
   TheFlag = FOUNDFLAG | DRCFLAG;
 
-  ResetConnections (false);
+  ResetConnections (false, false);
 
   /* Let the search stop if we find something we haven't yet seen */
   drc = true;
@@ -4038,7 +4038,7 @@ IsPolygonAnIsland (LayerType *layer, PolygonType *polygon)
   while (any_more);
 
   drc = false;
-  ResetConnections (false);
+  ResetConnections (false, false);
   FreeConnectionLookupMemory ();
 
   return (connected_count == 0);
